test/integration/e2e_test.go

```diff
@@ -111,12 +111,13 @@ func TestE2EFullIntegration_Browser(t *testing.T) {
 	)
 
 	// Create a JWTAuthenticator that will validate the tokens from the downstream issuer.
+	// if the FederationDomain is not Ready, the JWTAuthenticator cannot be ready, either.
```

Comment:
Looks like you meant to say "if" with a capital "I". I'm surprised the linter allowed this.

Reply:
👍